index.html

```diff
@@ -5,7 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="style.css" />
-    <title>Document</title>
+    <title>Portfolio-Project</title>
```

Comment:
- [ ] The page title should briefly describe the content of the page, please consider changing it to a meaningful title. For example:
  -  Fabien Portfolio